Orange/widgets/visualize/tests/test_owpythagorastree.py

```diff
@@ -0,0 +1,63 @@
+import unittest
+import math
+
+from Orange.widgets.visualize.owpythagorastree import \
+    PythagorasTree, Point, Square
+
+
+class TestPythagorasTree(unittest.TestCase):
+    def setUp(self):
+        self.builder = PythagorasTree()
+
+    def test_get_point_on_square_edge_with_no_angle(self):
+        point = self.builder._get_point_on_square_edge(
+            center=Point(0, 0), length=2, angle=0
+        )
+        expected_point = Point(1, 0)
+        self.assertAlmostEqual(point.x, expected_point.x, places=1)
+        self.assertAlmostEqual(point.y, expected_point.y, places=1)
+
+    def test_get_point_on_square_edge_with_angle_1(self):
+        point = self.builder._get_point_on_square_edge(
+            center=Point(3.06, 2.79), length=2.39, angle=math.radians(30)
+        )
+        expected_point = Point(4.08, 3.41)
+        self.assertAlmostEqual(point.x, expected_point.x, places=1)
+        self.assertAlmostEqual(point.y, expected_point.y, places=1)
+
+    def test_get_point_on_square_edge_with_angle_2(self):
+        point = self.builder._get_point_on_square_edge(
+            center=Point(2.7, 2.77), length=1.65, angle=math.radians(20.97)
```

Comment:
These three tests all seem to test `_get_point_on_square_edge`. They could be merged into a single test. And human-interpretable values like `(0, 0)`, `1`, `pi/2` should probably be preferred.